.github/workflows/macM1-e2e.yaml

```
@@ -0,0 +1,58 @@
+name: e2e tests on Mac M1
+
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: '0 9 * * 1-5'
+
+
+jobs:
+
+  e2e-tests:
+    timeout-minutes: 45
+    runs-on: arm64
```

```suggestion
    runs-on: [self-hosted, macos-latest, arm64]
```
Something needs to indicate that this is (for now) mac-only, and that we need to filter out github-hosted runners.